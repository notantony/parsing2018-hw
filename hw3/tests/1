f =
= a 10 if > 2 3 print 3 if > 4 - 7 2 print + 3 4