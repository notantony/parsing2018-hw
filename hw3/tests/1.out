void f() {
    int a;
    a = 10;
    if ((2 > 3)) {
        std::cout << 3;
    }
    else {
        if ((4 > (7 - 2)))
            std::cout << (3 + 4);
    }
}
